src/99-appendices/05-units.md

```diff
@@ -1,39 +1,68 @@
+As described in the [units section](../02-common-principles.md#units),
+the specification of units MUST follow the
+[International System of Units](https://en.wikipedia.org/wiki/International_System_of_Units)
+(SI, abbreviated from the French Système international (d'unités)).
 
-| Unit name      | Unit symbol | Quantity name                              |
-| ------------------------- | ------------------------------ | ------------------------------------------------------------------ |
-| metre          | m           | length                                     |
-| kilogram       | kg          | mass                                       |
-| second         | s           | time                                       |
-| ampere         | A           | electric current                           |
-| kelvin         | K           | thermodynamic temperature                  |
-| mole           | mol         | amount of substance                        |
-| candela        | cd          | luminous intensity                         |
-| radian         | rad         | angle                                      |
-| steradian      | sr          | solid angle                                |
-| hertz          | Hz          | frequency                                  |
-| newton         | N           | force, weight                              |
-| pascal         | Pa          | pressure, stress                           |
-| joule          | J           | energy, work, heat                         |
-| watt           | W           | power, radiant flux                        |
-| coulomb        | C           | electric charge or quantity of electricity |
-| volt           | V           | voltage (electrical potential), emf        |
-| farad          | F           | capacitance                                |
-| ohm            | Ω           | resistance, impedance, reactance           |
-| siemens        | S           | electrical conductance                     |
-| weber          | Wb          | magnetic flux                              |
-| tesla          | T           | magnetic flux density                      |
-| henry          | H           | inductance                                 |
-| degree Celsius | °C          | temperature relative to 273.15 K           |
-| lumen          | lm          | luminous flux                              |
-| lux            | lx          | illuminance                                |
-| becquerel      | Bq          | radioactivity (decays per unit time)       |
-| gray           | Gy          | absorbed dose (of ionizing radiation)      |
-| sievert        | Sv          | equivalent dose (of ionizing radiation)    |
-| katal          | kat         | catalytic activity                         |
+The [CMIXF-12](https://people.csail.mit.edu/jaffer/MIXF/CMIXF-12) convention
+for encoding units is RECOMMENDED to achieve maximum portability and limited
+variability of representation.
```

Comment:
```suggestion
variability of representation.
In case that a CMIXF representation of SI units is not possible, you will have
to declare your units as custom units and define them in accompanying JSON
files, as described in the [units section](../02-common-principles.md#units).
```